app/src/main/java/org/kiwix/kiwixmobile/zim_manager/local_file_transfer/TransferProgressFragment.java

```diff
@@ -34,8 +35,19 @@ public TransferProgressFragment() {
     // Required empty public constructor
   }
 
-  public TransferProgressFragment(ArrayList<FileItem> fileItems) {
-    this.fileItems = fileItems;
+  public static TransferProgressFragment newInstance(ArrayList<FileItem> fileItems) {
+    TransferProgressFragment fragment = new TransferProgressFragment();
+    Bundle args = new Bundle();
+    args.putParcelableArrayList("FILE_ITEMS", fileItems);
```

Comment:
extract a constant for "FILE_ITEMS"

Reply:
[Done](https://github.com/kiwix/kiwix-android/pull/1276/commits/9a2bee7732cb749d9387e139f97268599cb8b689#diff-a0f57270aa23e27cd4cd2d0cf244cb42R28)